### PR TITLE
#4 handle jing output correctly if _JAVA_OPTIONS env defined

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #4 

Any critics/feedback appreciated

P.S. @sshaw also I propose to add `.rubocop.yml` to the repo because the current code style is different from 'default' one + 3.1 ruby and ruby-head to test matrix. Let me know if this ok to make it in the scope of this PR

### CODE REVIEW

File: lib/jing.rb [hunk 1]
```diff
@@ -129,13 +128,18 @@ def execute(options)
...
         errors << {
           :source  => $1,
           :line    => $2.to_i,
           :column  => $3.to_i,
           :message => $4
         }
+      when /Picked up _JAVA_OPTIONS: /
+        # ignore diagnostic message
+      else # There must have been a problem that was not schema related
+        raise ExecutionError, line
```
Comment (sshaw) [inline lib/jing.rb:142]: Maybe we should provide `output` instead of `line` so the exception does not exclude any additional lines/context?

Comment (CAMOBAP) [reply lib/jing.rb]: Agree, make sens

File: test/test_jing.rb [hunk 2]
```diff
@@ -113,6 +113,14 @@ def test_successful_exit_status_with_unparsable_output_raises_an_exception
     end
   end
 
+  def test_java_options_env
+    java_opts = ENV["_JAVA_OPTIONS"]
+    ENV["_JAVA_OPTIONS"] = "-Djava.io.tmpdir=#{Dir.tmpdir}"
+    errors = Jing.new(RNG_SCHEMA).validate(VALID_XML)
```
Comment (sshaw) [inline test/test_jing.rb:119]: Maybe I'm missing something bug shouldn't this be ran under `fakeshell`, setting `:output` to something that contains the warning message?

Comment (CAMOBAP) [reply test/test_jing.rb]: Totaly make sense, done

**Comment by sshaw** [comment]
Released as v0.0.3 (though I see GitHub now includes that on this PR :octocat:)

**Pull request merged**